bedrock/base/templates/base-protocol.html

```diff
@@ -138,9 +138,13 @@
       <!--[if !IE]><!-->
         {# Standard site JS bundles served to all evergreen browsers plus IE 10 and above. #}
         {{ js_bundle('lib') }}
-        {{ js_bundle('ui') }}
         {{ js_bundle('fxa') }}
         {{ js_bundle('data') }}
+        {% if switch('m24-navigation-and-footer')and LANG.startswith('en-') %}
```

Comment:
Nit: space after brace
```suggestion
        {% if switch('m24-navigation-and-footer') and LANG.startswith('en-') %}
```